### PR TITLE
Generate README.md with helm-docs

### PR DESCRIPTION
**Pull request by babykart**
Related to #9
Actually, only the main `values.yaml` generates a proper `README.md`

```shell
helm-docs -t README.md.gotmpl
```

### CODE REVIEW

**Comment by babykart** [comment]
Feel free to update the comments.
I think that the good approach is to generate these README.md in the release pipeline.

**Comment by anbraten** [comment]
@babykart So should we add a ci step that automatically updates the `README.md` on release and pushes the changes to main?

File: Chart.yaml [hunk 1]
```diff
@@ -18,15 +18,16 @@ keywords:
...
+  - kubernetes
 
 sources:
-  - https://github.com/woodpecker-ci/woodpecker
-  - https://github.com/woodpecker-ci/helm
+  - https://github.com/woodpecker-ci/woodpecker.git
+  - https://github.com/woodpecker-ci/helm.git
 
 dependencies:
   - name: server
```
Comment (6543) [inline Chart.yaml:28]: Unrelated change

**Comment by babykart** [comment]
> @babykart So should we add a ci step that automatically updates the `README.md` on release and pushes the changes to main?

Yes :)
